bin/mirgecompare.py

```diff
@@ -28,7 +28,11 @@
     file_split = os.path.splitext(first_file)[1]
     file_type = file_split[1:]  # remove dot
 
-    user_tolerance = 1e-12
+    user_tolerance = {
+    "u": 1e-12,
+    "v": 1e-12
+    }
+
```

Comment:
What happens when a fieldname is not in the dict? Maybe add a function `get_tolerance(fieldname)` that returns either a value from the dict or another default value.

Reply:
Can this be a nested function or should it be defined outside of compare_files_vtu so it can be used with other compare functions?

Reply:
Sorry, it is here:

https://github.com/illinois-ceesd/mirgecom/blob/refactor-ux/mirgecom/simutil.py

Reply:
hmm, since this is not in main, what do we do? should we just add the function as part of this PR? Also, I was not sure what we do regarding the config_object. Is there an example on how to use it?

Reply:
I just wanted to bounce this function off of you guys since you were discussing adding a function to do the same thing.   It would be my preference to add a function like this to `simutil` and then use it in this utility.   I have other (WIP) uses for this function.

Regarding `config_object` - it is totally users choice.   A dictionary works in this function (https://github.com/illinois-ceesd/mirgecom/blob/23d4d4bab885b891e3d59fd14006ad74092f1e03/mirgecom/simutil.py#L465), but also other sorts of objects too, since `object.__dict__` is more general.   This API allows us to change what we use as the `config_object` without changing the function with which we extract data from it.